docker-compose.yml

```diff
@@ -21,3 +21,15 @@ services:
     volumes:
       - .:/application
     restart: unless-stopped
+  
+  node:
+    image: node:16-alpine3.14
+    container_name: intern-node
+    working_dir: /application
+    user: ${CURRENT_UID}
+    volumes:
+      - .:/application
+    tty: true
+    entrypoint: "./environment/node-entrypoint.sh"
```

Comment:
I think it's unneccessary. There's no file like that in our codebase.
```suggestion
```

Reply:
It needs to be created.

Reply:
Oh, wait, I can see that @mtracz already addressed this thing.

Reply:
I created this file, but due conversation on slack I reversed the changes and deleted thie file. Thanks for letting me know